package.json

```diff
@@ -8,7 +8,7 @@
     "dist"
   ],
   "scripts": {
-    "build": "npm run build:dev && npm run build:prod",
+    "build": "npm run build:dev && npm run build:prod && cp static/* dist",
```

Comment:
I suggest to make them part of the webpack process, so they are also served on dev.

Reply:
See 9e5aac2e884ebdbb337d6d7d26b2851f181a9f3c.